resource_customizations/numaplane.numaproj.io/PipelineRollout/actions/unpause-fast/action.lua

```diff
@@ -0,0 +1,6 @@
+obj.spec.pipeline.spec.lifecycle.desiredPhase = "Running"
+if obj.metadata.annotations == nil then
+    obj.metadata.annotations = {}
+end
+obj.metadata.annotations["numaflow.numaproj.io/resume-strategy"] = "fast"
```

Comment:
We need to actually set the metadata of the Pipeline definition, not the PipelineRollout, which is at `obj.spec.pipeline.spec.metadata`